terraform/files/bin/create_cluster.sh

```diff
@@ -166,7 +166,8 @@ if test "$USE_CILIUM" = "true" -o "${USE_CILIUM:0:1}" = "v"; then
   if test "${USE_CILIUM:0:1}" = "v"; then
     CILIUM_VERSION="${USE_CILIUM}"
   fi
-  KUBECONFIG=${KUBECONFIG_WORKLOADCLUSTER} cilium install --version $CILIUM_VERSION
+  POD_CIDR=$(yq eval '.POD_CIDR' $CCCFG)
+  KUBECONFIG=${KUBECONFIG_WORKLOADCLUSTER} cilium install --version $CILIUM_VERSION --helm-set-string ipam.operator.clusterPoolIPv4PodCIDRList=${POD_CIDR},
```

Comment:
`--helm-set-string ipam.operator.clusterPoolIPv4PodCIDRList=${POD_CIDR},` Does this comma at the end works?

According to the [Helm Docs](https://helm.sh/docs/intro/using_helm/#the-format-and-limitations-of---set), the list is defined inside curly brackets.